code/modules/shuttle/on_move.dm

```diff
@@ -299,6 +300,17 @@ All ShuttleMove procs go here
 	if (oldT && !is_reserved_level(oldT))
 		unlocked = TRUE
 
+/obj/item/gun/lateShuttleMove(turf/oldT, list/movement_force, move_dir)
```

Comment:
i'm not super comfortable with firing these procs directly from shuttle movement, but at least for the items themselves, there's not any other easy way. i would still lower the chance down from HIGH (even given the percentage chance changes i mentioned before) -- it should be pretty uncommon for guns to fire JUST because they were unsafetied during takeoff; it also strikes me as somewhat unnecessary to search all items' contents just for guns that might go off. it feels like an anti-cheese measure, but i'd argue that this is a flavor feature, not a balance one; i don't expect people to TRY to cheese it.

Reply:
I mainly had it there for consistency, but Im happy to cut it out. Content check has been removed.